oat-sa/extension-tao-itemqti
# Feature/aut 718/enable scrollable double column

**Related to:** https://oat-sa.atlassian.net/browse/AUT-718

**Description:** Add option to enable scrollable double-column layout.

**How to test:**

1. In `config/taoQtiItem/qtiCreator.conf.php` add:
 ```
'scrollable-multi-column' => true
 ```
2. Go to the item authoring
3. Create item
4. Open item Style Editor
5. Check `Scrollable multi-column` checkbox in `Layout` section
6. Check that `.grid-row` element contains `dual-column-layout` css class (see [comment](https://oat-sa.atlassian.net/browse/TRN-930?focusedCommentId=141949))
7. Save the item
8. Reload the page or return to manage items, etc.
9. Open the item again
10. Check that `Scrollable multi-column` checkbox is checked and css class is present

**Deployed on:** http://test-anastasia.playground.kitchen.it.taocloud.org:44451
